examples/src/main/scala/org/platanios/tensorflow/examples/python2scala/LinearRegressionFromRestoredPythonModel.scala

```diff
@@ -0,0 +1,123 @@
+package org.platanios.tensorflow.examples.python2scala
+
+import java.io._
+import java.nio.file.Paths
+
+import com.typesafe.scalalogging.Logger
+import org.platanios.tensorflow.api.core.Graph
+import org.platanios.tensorflow.api.ops.Output
+import org.platanios.tensorflow.api.{Op, Session, Shape, Tensor, tf}
+import org.slf4j.LoggerFactory
+import org.tensorflow.framework.MetaGraphDef
+
+import scala.collection.mutable.ArrayBuffer
+import scala.util.Random
+
+/**
+  * @author Luca Tagliabue
+  *         Purpose: take a model trained with the Python API of Tensor Flow and restore
+  *         it inside a Scala environment in order to continue the training process. The
+  *         model was saved using tf.train.Saver()
+  *
+  *         Version used:
+  *         TensorFlow version: 1.8.0rc1
+  *         ScalaTensorFlow version: 0.2.0-SNAPSHOT
+  */
+object LinearRegressionFromRestoredPythonModel {
+  private[this] val logger = Logger(LoggerFactory.getLogger("Examples / Linear Regression"))
+  private[this] val random = new Random(22)
+  private[this] val weight = random.nextFloat()
+
+  def main(args: Array[String]): Unit = {
+    val meta = "examples/src/main/scala/org/platanios/tensorflow/examples/python2scala/virgin-linear-regression-pull-request.meta"
+    val checkpoint = "examples/src/main/scala/org/platanios/tensorflow/examples/python2scala/virgin-linear-regression-pull-request"
+    val metaGraphDefFile = "examples/src/main/scala/org/platanios/tensorflow/examples/python2scala/MetaGraphDef.txt"
+
+    val metaGraphDefInputStream = new BufferedInputStream(new FileInputStream(meta))
+    val mgf = MetaGraphDef.parseFrom(metaGraphDefInputStream)
+    val checkpointPath = Paths.get(checkpoint)
+    scala.reflect.io.File(metaGraphDefFile).writeAll(mgf.toString)
```

Comment:
Why are you using `scala.reflect.io.File` and not the more standard java `File`?